src/compressed_tensors/quantization/observers/min_max.py

```diff
@@ -62,4 +64,19 @@ def calculate_qparams(self, observed: Tensor) -> Tuple[FloatTensor, IntTensor]:
 
         self.counter += 1
 
-        return calculate_qparams(min_val, max_val, self.quantization_args)
+        if self.quantization_args.symmetric:
```

Comment:
I think you'll need to rebase this - would be good to keep the logic going through calculate_qparams so it can easily be shared and expanded between observers

Reply:
Ah good catch, moved this logic over to calculate_qparams